docs/revsets.md

```diff
@@ -157,6 +157,25 @@ For example:
 'user(x)' = 'author(x) | committer(x)'
 ```
 
+### Built-in Aliases
+
+The following aliases are built-in and might be relied on to exist. In every case,
+these functions are defined as aliases to allow you to overwrite them in specific
+situations.
```

Comment:
You can put "See [aliases.toml](./cli/src/config/aliases.toml) for the exact definitions" here.